### DISCUSSION TITLE
How pgs-calc identify multialelic variants?

### DISCUSSION THREAD

**Discussion by agnis-iohana** [PGScatalog/pgsc_calc#102] [Q&A] [answered]
Hi! 
I want to calculate pgs for a trait with multialelic variant reported by literature, but I don't know how to include this type of SNP in my score file. How pgs-calc identify it in my vcf? Is there a way to standardize this information in my score file?
I tested a multialelic variant like this:
rsID    chr_name        chr_position    effect_allele   effect_weight   hm_source       hm_rsID hm_chr  hm_pos  hm_inferOtherAllele
rs63751012      3       37035147        A       0.066   ENSEMBL rs34579442      3       37035147        G,C,T

my vcf report this variant like this:
3       37035147        rs63751012      G       A,C,T   .       PASS    .       GT:GQ

But this SNP appears as a mismatch and the multialelic field is shown as false in the output, however I set --keep_multiallelic parameter as true.

Thanks!

**Comment by nebfield** [comment, marked as answer]
Thanks for your question and sorry for the late reply. The GitHub notification got lost in my emails 👻 

If I understand correctly you want to use a custom scoring file (not published in the PGS Catalog) that contains a variant with multiple other alleles.

We have some documentation about creating custom scoring files. The data structure is different to the example you described:

https://pgsc-calc.readthedocs.io/en/latest/how-to/calculate_custom.html#scorefile-setup

However, in `v1.3.2` we don't support variants with multiple `other alleles` encoded. These variants are removed internally during quality control steps. This is why the variant is labelled as not multi-allelic. Correctly handling multi-allelic variants can be complicated so this might change in a future release.

One potential workaround would be to test making a custom scoring file using only `effect_allele` data (leaving `other_allele` empty).

Thanks for your interest in `pgsc_calc` 🚀
